lib/data/US.yml

```diff
@@ -679,6 +679,31 @@
   :state_name: Puerto Rico
   :city: Juana Diaz
   :time_zone: America/Puerto_Rico
+'00802':
+  :state_code: VI
+  :state_name: Virgin Islands
+  :city: Saint Thomas
+  :time_zone: America/St_Thomas
+'00820':
+  :state_code: VI
+  :state_name: Virgin Islands
+  :city: Christiansted
+  :time_zone: America/St_Thomas
+'00830':
+  :state_code: VI
+  :state_name: Virgin Islands
+  :city: Saint John
+  :time_zone: America/St_Thomas
+'00840':
+  :state_code: VI
+  :state_name: Virgin Islands
+  :city: Kingshill
```

Comment:
According to the USPS, this city name should be Frederiksted

Reply:
Ah, yes. USPS. I didn't know this is the official one. Just confirmed that it indeed is FREDERIKSTED from the following lookup service.
https://tools.usps.com/zip-code-lookup.htm?citybyzipcode

Updated the PR. Thanks.